knowage-core/src/main/java/it/eng/spagobi/api/AbstractDocumentResource.java

```diff
@@ -194,7 +194,7 @@ public Response getDocumentTemplate(String label) {
 		return rb.build();
 	}
 
-	public Response addDocumentTemplate(String label, MultiPartBody input) {
+	protected Response addDocumentTemplate(String label, MultiPartBody input) {
```

Comment:
If possible, I would prefer private but this is an abstract class, we need a check.

Reply:
This one can't be private, it is being used in DocumentResource.java (two files, one of them is v2).